include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -326,6 +326,56 @@ def TTIR_ReluOp : TTIR_ElementwiseUnaryOp<"relu"> {
     }];
 }
 
+def TTIR_RoundOp : TTIR_DPSOp<"round"> {
+    let summary = "Eltwise round.";
+    let description = [{
+      Rounds a number to the nearest value away from zero.
+
+      Example:
+      // %operand = [-2.5, 0.4, 0.5, 0.6, 2.5]
+      %result = "stablehlo.round_nearest_afz"(%operand) : (tensor<5xf64>) -> tensor<5xf64>
+      // %result: [-3.0, 0.0, 1.0, 1.0, 3.0]
+    }];
+    let arguments = (ins
+        AnyRankedTensor:$input,
+        AnyRankedTensor:$output,
+        DefaultValuedAttr<I32Attr, "1">:$decimals
+    );
+
+    let extraClassDeclaration = [{
+      MutableOperandRange getDpsInitsMutable() { return getOutputMutable(); }
+    }];
+
+    let results = (outs AnyRankedTensor:$result);
+
+    let hasVerifier = 1;
+}
+
+def TTIR_RoundNearestEvenOp : TTIR_DPSOp<"roundnearesteven"> {
+    let summary = "Eltwise round towards nearest even.";
+    let description = [{
+      Rounds a number to the nearest value. If the number is exactly halfway between two values, the value is rounded to the nearest even value.
+
+      Example:
+      // %operand = [-2.5, 0.4, 0.5, 0.6, 2.5]
+      %result = "stablehlo.round_nearest_even"(%operand) : (tensor<5xf64>) -> tensor<5xf64>
+      // %result: [-2.0, 0.0, 0.0, 1.0, 2.0]
+    }];
+    let arguments = (ins
+        AnyRankedTensor:$input,
+        AnyRankedTensor:$output,
+        DefaultValuedAttr<I32Attr, "0">:$decimals
```

Comment:
What's the meaning of the `decimals` attribute? I would assume it's the number of significant digits after decimal point we are rounding to, but that doesn't look like the case here, so some explanation would be welcome. Same for the `TTIR_RoundOp`. It's also very questionable if we really need both of these ops, we are mostly dealing with low precision floating point types, I doubt the type of rounding will cause any significant change here.